### PR TITLE
Change png format to json file when downloading analysis results

### PR DESCRIPTION
**Pull request by scarletnguyen13**
JSON file sample:

Screen Shot 2020-07-30 at 3 57 40 PM

### CODE REVIEW

File: client/src/components/SentimentAnalysis/SentimentAnalysis.js [hunk 1]
```diff
@@ -81,6 +82,7 @@ const SentimentAnalysis = () => {
    * Set the current headline and content analysis of
    * the component with new data
    * @param {Boolean} data - new data returned from API or Firestore
+   * @param {ReactElement} form - current analysis form
```
Comment (rob-marcus) [inline client/src/components/SentimentAnalysis/SentimentAnalysis.js:85]: Looks like `data` is the only param?

File: client/src/components/SentimentAnalysis/SentimentAnalysis.js [hunk 2]
```diff
@@ -116,20 +118,17 @@ const SentimentAnalysis = () => {
   };
 
   /**
-   * Generates a PNG screenshot of the current result page and
+   * Generates a JSON file of the current result page and
```
Comment (rob-marcus) [inline client/src/components/SentimentAnalysis/SentimentAnalysis.js:121]: Minor nit: it's just creating a json file of the language analysis results right?

Comment (scarletnguyen13) [reply client/src/components/SentimentAnalysis/SentimentAnalysis.js:121]: yeah, and the content is included as well!

File: client/src/components/SentimentAnalysis/SentimentAnalysis.js [hunk 3]
```diff
@@ -116,20 +117,17 @@ const SentimentAnalysis = () => {
...
-    setDownloadStatus('downloading...');
-    const input = document.getElementById('analysis-container');
-    html2canvas(input).then(canvas => {
-      const imgData = canvas.toDataURL('image/png');
-      const uniqueId = new Date().getTime();
-      saveAs(imgData, `tardigrade-analysis-${uniqueId}.png`);
-      setTimeout(() => setDownloadStatus('downloaded'), 1000);
-    });
+  const downloadAnalysisJson = () => {
+    const indentationLevel = 2;
```
Comment (andrewlathamgoogle) [inline client/src/components/SentimentAnalysis/SentimentAnalysis.js:125]: No need to make this a separate variable.

**Pull request merged**